.github/workflows/ci.yml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x, 21.x]
+        node-version: [18.x, 22.x]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)